pkgs/tools/misc/man-db/default.nix

```diff
@@ -43,7 +65,15 @@ stdenv.mkDerivation rec {
     "ac_cv_func_mempcpy=no"
   ];
 
+
   preConfigure = ''
+    # need to recreate configure script due to substitutions in postPatch
+    ./bootstrap --gnulib-srcdir=${gnulib.src} --no-git
+
+    # deal with autoconf 2.70 bug: https://lists.gnu.org/archive/html/bug-autoconf/2020-12/msg00036.html
+    # can be removed once autoconf 2.71 is merged
+    patch < ${./fix-configure.patch}
```

Comment:
Could this be in patches?

Reply:
I guess not. This fixes the invalid configure script generated by autoconf which happens after the patch phase.